### PR TITLE
[3563] Incorrect message appears in ChangePasswordPage

### PR DESCRIPTION
**Pull request by BrunoScorzafave**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3563

**Problem:**
* Fields don't have validation rules
* Incorrect message appears in  ChangePasswordPage

**In this PR:**
* Create validation rules for password fields

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
1. Still can reproduce
Screenshot 2022-02-02 at 09 35 08 (2)

2. Field border is grey if user entered new password
Screenshot 2022-02-02 at 09 30 37

**Expected result:**
Field is green as in screenshot
Screenshot 2022-02-02 at 09 32 46

**Notes:**
New password validation happens only when submitting form (the same situation with error messages). Please return form validation as it was (it happens immediately)

**Comment by aleksandrakorolova** [comment]
Still, can reproduce

**Comment by aleksandrakorolova** [comment]
1. Still, can reproduce
2. Fixed
3. Error message 'Incorrect input!' appears below Confirm password field
Please return the previous error message 'Passwords do not match!' if user entered different passwords in New password and Confirm password fields
Screenshot 2022-02-22 at 10 19 48 (2)

**Comment by aleksandrakorolova** [comment]
1. Fixed 
now error message appears immediately below field
2. Fixed
3. Still can reproduce
Screenshot 2022-03-10 at 11 26 54

File: packages/scandipwa/src/component/MyAccountCreateAccount/MyAccountCreateAccount.config.js [hunk 1]
```diff
@@ -12,3 +12,4 @@
 export const SHOW_VAT_NUMBER_REQUIRED = 'req';
 export const SHOW_VAT_NUMBER_OPTIONAL = 'opt';
 export const CONFIRMATION_REQUIRED = 'confirmation_required';
+export const MIN_CHARACTER_SETS_IN_PASSWORD = 3;
```
Comment (riha112) [inline packages/scandipwa/src/component/MyAccountCreateAccount/MyAccountCreateAccount.config.js:15]: This is never used.

**Review by riha112 — CHANGES_REQUESTED** [review]

**Review by riha112 — APPROVED** [review]

**Pull request merged**